src/main/resources/application.yml

```diff
@@ -12,7 +12,7 @@ mod-lists:
       access-key: ${S3_ACCESS_KEY_ID:}
       secret-key: ${S3_SECRET_ACCESS_KEY:}
 server:
-  port: 8081
+  port: 8082
```

Comment:
This doesn't seem like something we should change

Reply:
@mweaver-ebsco , we are not considering this PR now. This is just rough work